src/store.js

```diff
@@ -34,8 +34,20 @@ store.dispatch(setContext('playlistContext', playlistContext));
 store.dispatch(setContext('courseContext', courseContext));
 store.dispatch(setContext('readmeContext', readmeContext));
 store.dispatch(setLessons(lessons));
-store.dispatch(setModeStudent());
 store.dispatch(setFilter(getTags(lessonContext, courseContext)));
-store.dispatch(setLanguage('nb'));
+
+/*localStorage*/
+if (localStorage.length === 0) {
+  store.dispatch(setMode(true));
+  store.dispatch(setLanguage('nb'));
+  store.dispatch(setWelcomeBox());
+}
+else {
+  store.dispatch(setMode(JSON.parse(localStorage.studentMode)));
+  store.dispatch(setLanguage(localStorage.lastLanguage));
+  JSON.parse(localStorage.welcomeBox) ? store.dispatch(setWelcomeBox()) : store.dispatch(setButton());
```

Comment:
I stedet for å ha både `setWelcomeBox` og `setButton`, hadde det ikke vært lettere å velge én av dem og gi den true/false, f.eks.`setWelcomeBox(true/false)` ? Så kan du kutte ut en ACTION type (SET_BUTTON), og heller sende inn en payload...?